release-notes/5.0/api-diff/netstandard2.1/5.0_System.Security.Cryptography.X509Certificates.md

```diff
@@ -0,0 +1,52 @@
+# System.Security.Cryptography.X509Certificates
+
+``` diff
+ namespace System.Security.Cryptography.X509Certificates {
+     public sealed class CertificateRequest {
++        public X509Certificate2 Create(X500DistinguishedName issuerName, X509SignatureGenerator generator, DateTimeOffset notBefore, DateTimeOffset notAfter, ReadOnlySpan<byte> serialNumber);
```

Comment:
I can understand "last release to this release" and "last LTS to this release" (they happen to both be from 3.1 to 5.0 in this case); but why would we publish "from two LTSes ago"?  That's just "2.1 -> 3.1 union 3.1 -> 5.0"

Reply:
cc @richlander